### PR TITLE
Pkg 5444

### PR DESCRIPTION
**Pull request by cbouss**
popt 1.19

**Destination channel:** main

### Links

- https://anaconda.atlassian.net/browse/PKG-5444
- https://github.com/rpm-software-management/popt

### Explanation of changes:

- For rsync

### CODE REVIEW

**Review by ryanskeith — APPROVED** [review]
LTGM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,10 +40,13 @@ test:
     - test -f $PREFIX/lib/libpopt.dylib  # [osx]
 
 about:
-  home: http://rpm5.org/
+  home: https://rpm5.org/
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  home: https://rpm.org/
```
If you remove `ftp.` from the source url `http://ftp.rpm.org/` you will get the home url.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+{% set version = "1.19" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - url: ftp://anduin.linuxfromscratch.org/BLFS/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: e728ed296fe9f069a0e005003c3d6b2dde3d9cad453422a10d6558616d304cc8
+  - url: https://ftp.osuosl.org/pub/rpm/{{ name }}/releases/{{ name }}-1.x/{{ name }}-{{ version }}.tar.gz
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The dev_url https://github.com/rpm-software-management/popt points to http://ftp.rpm.org/popt/. 
Should the source url be taken from http://ftp.rpm.org/popt/releases/popt-1.x/?
https://osuosl.org/ is an URL of Oregon State University Open Source Lab.

How can we verify that http://ftp.rpm.org is identical to OSUOSL? It looks like a mess. 

In other words, what is a canonical URL?

Comment (cbouss) [reply recipe/meta.yaml]: That's the url referenced in the release note at least: https://github.com/rpm-software-management/popt/releases

_[popt 1.19](https://github.com/rpm-software-management/popt/releases/tag/popt-1.19-release) [Latest](https://github.com/rpm-software-management/popt/releases/latest)
Download
https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
SHA256SUM: c25a4838fc8e4c1c8aacb8bd620edb3084a3d63bf8987fdad3ca2758c63240f9_

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: Updated to ftp.rpm.org for source and rpm.org for home. Thanks!

File: recipe/meta.yaml (hunk 2 above)
Comment (cbouss) [reply recipe/meta.yaml]: I updated to ftp.rpm.org in the end. Thanks!

**Review by SC426 — APPROVED** [review]

**Pull request merged**